language-server/modules/langserver-core/src/test/resources/completion/new/function_def/config/config15.json

```diff
@@ -471,28 +471,14 @@
       "insertText": "testParam2",
       "insertTextFormat": "Snippet"
     },
-    {
```

Comment:
For the following source snippet,

```
import ballerina/module1;

function testFunction1(int testParam1, int testParam2 = t<cursor>)

function testFunction() returns int {
    return 1;
}

public type TestRecord1 record {
    int field1 = 1;
    int field2 = 2;
};
```

In the mentioned cursor position, we do not get the `testFunction` as a visible symbol.

Reply:
```
import ballerina/module1;

function testFunction1(int testParam1, int testParam2 = t<cursor> INVALID[)]

function  INVALID[testFunction]() returns int {
    return 1;
}
 MISSING[)] MISSING[{] MISSING[}]
public type TestRecord1 record {
    int field1 = 1;
    int field2 = 2;
};
``` 
With this change `testFunction` take as an invalid node. Isn't this the problem?

Reply:
`testFunction` should not identify as `typeDescriptor` here. It should be kept as it is and the `testFunction1`'s recovery should be handled properly.